src/org/zaproxy/zap/extension/jxbrowserlinux32/ExtensionJxBrowserLinux32.java

```diff
@@ -58,6 +58,11 @@ public void init() {
     }
 
     @Override
+    public int getOrder() {

```

Comment:
The order should no longer be needed if the `ExtensionSelenium` provides the combo box model with available configured browsers(?).